Dockerfile

```diff
@@ -0,0 +1,23 @@
+# An image of ducktape that can be used to setup a Docker cluster where ducktape is run inside the container.
+
+FROM ubuntu:14.04
+
+RUN apt-get update && \
+    apt-get install -y libffi-dev libssl-dev openssh-server python-dev python-pip python-virtualenv && \
+    virtualenv /opt/ducktape && \
+    . /opt/ducktape/bin/activate && \
+    pip install -U pip==9.0.3 setuptools wheel && \
+    pip install bcrypt cryptography==2.2.2 pynacl && \
```

Comment:
parameterize pip and cryptography version

Reply:
Remove pip version as we don't need it in virtualenv. As for cryptography, we don't actually need to pin the version, but due to 14.04, there is a warning to upgrade to Python 2.7.7. Maybe later when we support different OS versions.